app/javascript/crayons/Modal/__tests__/Modal.test.jsx

```diff
@@ -101,7 +101,7 @@ it('should close on click outside, if enabled', () => {
       <Modal
         title="This is a modal title"
         onClose={onClose}
-        closeOnClickOutside
+        backdropDismissible
```

Comment:
Refactor (blocking): Rename to make the intention clearer.

```suggestion
        onBackdropDismissible
```